{"name": "Если передать неверный id заказа, запрос вернёт ошибку", "status": "passed", "description": "Запрос на принятие заказа с несуществующим id заказа возвращает код ответа 404 Not Found и \"message\":  \"Заказа с таким id не существует\"", "steps": [{"name": "Отправляем запрос принять заказ с несуществующим id заказа", "status": "passed", "steps": [{"name": "Принимаем заказ", "status": "passed", "parameters": [{"name": "order_id", "value": "'9999999'"}, {"name": "courier_id", "value": "607833"}], "start": 1758048394968, "stop": 1758048395622}], "start": 1758048394967, "stop": 1758048395622}], "start": 1758048394967, "stop": 1758048395625, "uuid": "6ac47a24-c432-45e2-992f-03a9ffb87ed2", "historyId": "010ac9f49caa51c54e88f908d076fe86", "testCaseId": "010ac9f49caa51c54e88f908d076fe86", "fullName": "tests.test_order_accept.TestAcceptOrder#test_do_not_accept_order_with_nonexistent_order_id", "labels": [{"name": "subSuite", "value": "class TestAcceptOrder: Принять заказ"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_accept"}, {"name": "host", "value": "MacBook-Pro-Ivan.local"}, {"name": "thread", "value": "81886-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_accept"}], "titlePath": ["tests", "test_order_accept.py", "TestAcceptOrder"]}